Lib/asyncio/tasks.py

```diff
@@ -116,6 +116,10 @@ def all_tasks(cls, loop=None):
         return _all_tasks_compat(loop)
 
     def __init__(self, coro, *, loop=None, name=None):
+        if loop:
+            warnings.warn("The loop argument is deprecated since Python 3.8, "
```

Comment:
`Task` is called with explicit `loop` by `loop.create_task()` method, please drop the check